### ISSUE TITLE
More clear instructions on Part 6 (and maybe a heads up on the programming language?)

### ISSUE THREAD

**Issue by hrose6967** [Gerzer/Git-Exam#1]
I found part 6 (the UICS.py) in the 5th step was rather confusing in the directions. They were long, wordy, confusing and were overall hard to read at times. While I understand it is hard to cut down on the instructions, it would be easier to read if the directions inside the UICS.py file were cutoff at about 100 characters per line. That way they fit on most people's screens and we don't have to either switch files or scroll right to see them.

As someone who doesn't know python, doing parts 1-6 was more of a hassle to quickly learn python. This could've been avoid if it was announced before the exam that we would have to code in python. That way learning could be done before the exam and not during the exam.

**Comment by LittleSilver33** [comment]
I second this so much. It is unrealistic to expect every student to know how to code in Python. Perhaps include the code edits we need to make in the instructions. It is not testing our coding, just our git knowledge.

**Comment by crizbae** [comment]
I agree. I see no purpose in having us write code for the git exam. I think the quality of the exam would be unchanged in fact improved if all of the code we needed to write was available elsewhere to copy-paste besides the trivial questions of changing a return value to a different string.

**Comment by Gerzer** [comment]
Thanks for the feedback! CSCI-1100 Computer Science I, which teaches Python, is a prerequisite for RCOS, so all students are indeed expected to know how to code in Python. However, you’re right that that’s not the primary purpose of the exam. The programming tasks were designed to be short and easy for anyone who has the expected Python skills, but I’ll leave this issue open so that RCOS leadership can consider the suggestion that the solution code be provided directly.

**Comment by wdturner** [comment]
I hate to do this, but if we are going to work on this with a PR, we should make the first and separate paragraphs separate issues. Can you submit the first paragraph separately? All of the comments seem to be about the use of Python, so we will make this issue about that and open a separate one for the confusing text.
